tracing/trace_state.cc

```diff
@@ -55,15 +55,19 @@ namespace tracing {
 logging::logger trace_state_logger("trace_state");
 
 struct trace_state::params_values {
+    struct prepared_statement_info {
+        prepared_checked_weak_ptr statement;
+        std::optional<std::vector<sstring_view>> query_option_names;
+        std::vector<cql3::raw_value_view> query_option_values;
```

Comment:
You probably want to introduce a constructor that receives `prepared_checked_weak_ptr`. See more below.